contracts/contracts/UserReplicaSetManager.sol

```diff
@@ -73,13 +74,26 @@ contract UserReplicaSetManager is SigningLogicInitializable, RegistryContract {
         registry = RegistryInterface(_registryAddress);
         userFactoryRegistryKey = _userFactoryRegistryKey;
         userReplicaSetBootstrapAddress = _userReplicaSetBootstrapAddress;
-        _seedBootstrapNodes(_bootstrapSPIds, _bootstrapNodeDelegateWallets);
         // Initialize base Signing Logic contract
         SigningLogicInitializable.initialize(
             "User Replica Set Manager",
             "1",
             _networkId
         );
+        seedComplete = false;
+    }
+
+    /**
+     * @notice Function used to initialize bootstrap nodes
+     * Prior to this function, the contract is effectively disabled
+     */
+    function seedBootstrapNodes(
+        uint[] calldata _bootstrapSPIds,
+        address[] calldata _bootstrapNodeDelegateWallets
+    ) external
+    {
+        require(seedComplete == false, "Seed operation already completed");
+        _seedBootstrapNodes(_bootstrapSPIds, _bootstrapNodeDelegateWallets);
```

Comment:
nitty but no need to hav separate internal function if its only called once from here